tests/testthat/test-learn.R

```diff
@@ -1,49 +1,87 @@
 # FIXME: #313: learn only from current source
 
-test_that("Learning from MSSQL works?", {
-
+test_that("Standard learning from MSSQL (schema 'dbo') works?", {
 
+  skip_if_src_not("mssql")
   # dm_learn_from_mssql() --------------------------------------------------
-  src_mssql <- skip_if_error(src_test("mssql"))
+  src_mssql <- my_test_src()
 
   # create an object on the MSSQL-DB that can be learned
   if (!any(src_tbls(src_mssql) %>%
     grepl("^tf_1_", .))) {
     copy_dm_to(src_mssql, dm_for_filter(), temporary = FALSE, table_names = unique_db_table_name)
   }
 
-  dm_for_filter_mssql_learned <- dm_from_src(src_mssql)
+  dm_for_filter_mssql_learned_all <- dm_from_src(src_mssql)
 
-  def_learned_renamed_reclassed <-
-    dm_rename_tbl(
-      dm_for_filter_mssql_learned,
-      structure(src_tbls(dm_for_filter_mssql_learned), names = src_tbls(dm_for_filter()))
-    ) %>%
-    dm_get_def() %>%
-    select(-data)
-
-  def_original <-
-    dm_get_def(dm_for_filter()) %>%
-    select(-data)
-
-  expect_identical(
-    def_learned_renamed_reclassed,
-    def_original
+  # in case there happen to be other tables in schema "dbo"
+  dm_for_filter_mssql_learned <- dm_select_tbl(
+    dm_for_filter_mssql_learned_all,
```

Comment:
When using the pipe, please start with the first argument. Mixing piped and non-piped code is confusing.

```suggestion
  dm_for_filter_mssql_learned <-
    dm_for_filter_mssql_learned_all %>%
    dm_select_tbl(
```